packages/components/pagination/src/pagination.ts

```diff
@@ -186,6 +193,17 @@ export default defineComponent({
     const { t } = useLocale()
     const ns = useNamespace('pagination')
     const vnodeProps = getCurrentInstance()!.vnode.props || {}
+    const _size = computed(() => (props.small ? 'small' : props?.size))
```

Comment:
A default value should be given to ensure that the default `class` is consistent with the previous one.
```suggestion
    const _size = computed(() => (props.small ? 'small' : props.size || 'default'))
```

Reply:
I don't think it's necessary to add this, because I observe that the component's style does not extract 'default' to write, so no setting is the default, you think

Reply:
Yes, there is indeed no style for `default`, but there is a corresponding class for `default`. I think it's best to keep it consistent with before.
@kooriookami WDYT?

Reply:
Other components set the default value to '', e.g. `props.size || ''`.

Reply:
Well, keep it consistent with the other components.